src/table.jl

```diff
@@ -46,6 +46,8 @@ end
 getshp(t::Table) = getfield(t, :shp)
 getdbf(t::Table) = getfield(t, :dbf)
 
+getshp(t::Row) = getfield(t, :geometry)
```

Comment:
There is already the `shape` function below that does the same thing, so this isn't needed.

Reply:
Removed this